src/Momento.Sdk.Incubating/SimpleCacheClient.cs

```diff
@@ -206,11 +206,11 @@ public async Task<CacheDictionarySetResponse> DictionarySetAsync(string cacheNam
             return new CacheDictionarySetResponse.Error(new InvalidArgumentException(e.Message));
         }
 
-        return await this.dataClient.DictionarySetAsync(cacheName, dictionaryName, field, value, ttl);
+        return await this.dataClient.DictionarySetFieldAsync(cacheName, dictionaryName, field, value, ttl);
     }
 
     /// <inheritdoc cref="DictionarySetAsync(string, string, byte[], byte[], CollectionTtl)"/>
```

Comment:
The code references here and elsewhere need to be updated with the latest names as well.